pkgs/top-level/aliases.nix

```
@@ -2052,7 +2052,7 @@ mapAliases {
     'wayfireApplications-unwrapped.wlroots' has been removed
   ''; # Add 2023-07-29
   waypoint = throw "waypoint has been removed from nixpkgs as the upstream project was archived"; # Added 2024-04-24
-  webkitgtk = lib.warnOnInstantiate "Explicitly set the ABI version of 'webkitgtk'" webkitgtk_4_0;
+  webkitgtk = throw "webkitgtk has been removed from nixpkgs, set the ABI version explicitly"; # Added 2025-06-11
```

Probably should be quoted since otherwise it soundsd like we removed all instances of WebkitGTK.

```suggestion
  webkitgtk = throw "'webkitgtk' attribute has been removed, set the ABI version explicitly"; # Added 2025-06-11
```

---

```suggestion
  webkitgtk = throw "'webkitgtk' attribute has been removed from nixpkgs, use attribute with ABI version set explicitly"; # Added 2025-06-11
```

---

Added suggestion by @Aleksanaa and both as co-authored-by